src/components/shared/Tab.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+   <div class="tab-image-wrapper" @click="$emit('tab-clicked')">
+    <img class="tab-image" :src="tab.imageSrc" alt="image" />
+    <div class="tab-image-wrapper-overlay" :class="`${activeTab !== tab.name && 'overlay'}`" />
+   </div>
+</template>
+
+<script>
+export default {
+  // eslint-disable-next-line vue/multi-word-component-names
+  name: 'Tab',
```

Comment:
Is this component truly "shared"? The name 'Tab' is kinda vague. Especially if you have images/overlays inside which will define it as a pokemon image or something like that.